manifests/config.pp

```diff
@@ -16,7 +16,7 @@
           ensure  => directory,
           owner   => 0,
           group   => 0,
-          mode    => '0664',
+          mode    => '0775',
```

Comment:
Yep, which is why it works even with '0664' but it'd be nice for the mode to actually be accurate.